src/azure-cli/azure/cli/command_modules/appservice/_params.py

```diff
@@ -1103,6 +1103,14 @@ def load_arguments(self, _):
     with self.argument_context('staticwebapp functions link') as c:
         c.argument('function_resource_id', help="Resource ID of the functionapp to link. Can be retrieved with 'az functionapp --query id'")
         c.argument('force', help="Force the function link even if the function is already linked to a static webapp. May be needed if the function was previously linked to a static webapp.")
+    with self.argument_context('staticwebapp backends link') as c:
+        c.argument('backend_resource_id', help="Resource ID of the backend to link.")
+        c.argument('backend_region', help="Region the backend resource lives in.")
+    with self.argument_context('staticwebapp backends validate') as c:
+        c.argument('backend_resource_id', help="Resource ID of the backend to link.")
+        c.argument('backend_region', help="Region the backend resource lives in.")
+    with self.argument_context('staticwebapp backends unlink') as c:
+        c.argument('is_cleaning_auth_config', help="Removed the configures auth provider from the backend and disables auth if no other provider configured.")
```

Comment:
Maybe something like "If set to true, removes the identity provider configured on the backend during the linking process."